material_theme_builder/android/app/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,30 @@
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    package="com.example.material_theme_builder">
```

Comment:
This is a nit, but I'm trying to keep the package names for samples relatively cohesive. This should be "dev.flutter.examples", and you can do that update with this command from the project root:

    flutter create --org=dev.flutter.examples .

Reply:
Done.